### PR TITLE
RHMAP-2594 Drop to Android 22

### PR DESCRIPTION
**Pull request by david-martin**

### CODE REVIEW

**Comment by david-martin** [comment]
@wei-lee is this the only change needed to drop to android 22 (and remove the need for jdk7)?

File: app/build.gradle [hunk 1]
```diff
@@ -5,7 +5,7 @@ plugins {
 apply plugin: 'com.android.application'
 
 android {
-    compileSdkVersion 23
+    compileSdkVersion 22
     buildToolsVersion "23.0.1"
```
Comment (wei-lee) [inline app/build.gradle]: you need to drop this to 22.0.1 as well. See https://github.com/feedhenry-templates/saml-android-app/blob/master/app/build.gradle

**Comment by danielpassos** [comment]
@david-martin No, it's not. This project is using some resource from API Level 23, only rollback the version for 22 will broke the app. I'm working on it [here](https://github.com/danielpassos/sync-android-app/tree/improvements)

**Comment by david-martin** [comment]
@danielpassos thanks, i'll wait for another pr to be merged before rebuilding our templates

**Pull request closed without merging**